docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md

```diff
@@ -295,149 +298,27 @@ On each node, perform the following as a user with sudo access:
-
-      ```sh
-      cd /usr/local
-      sudo tar xvfz path-to-s3cmd-2.0.1.tar.gz
-      sudo ln -s /usr/local/s3cmd-2.0.1/s3cmd /usr/local/bin/s3cmd
-      ```
+## Enable yugabyte user processes to run after logout
 
-- Azure Storage: Install azcopy using one of the following options:
-
-  - Download `azcopy_linux_amd64_10.13.0.tar.gz` using the following command:
-
-      ```sh
-      wget https://azcopyvnext.azureedge.net/release20211027/azcopy_linux_amd64_10.13.0.tar.gz
-      ```
-
-  - For airgapped installations, copy `/opt/third-party/azcopy_linux_amd64_10.13.0.tar.gz` from the YugabyteDB Anywhere node, as follows:
-
-      ```sh
-      cd /usr/local
-      sudo tar xfz path-to-azcopy_linux_amd64_10.13.0.tar.gz -C /usr/local/bin azcopy_linux_amd64_10.13.0/azcopy --strip-components 1
-      ```
-
-- Google Cloud Storage: Install gsutil using one of the following options:
-
-  - Download `gsutil_4.60.tar.gz` using the following command:
-
-      ```sh
-      wget https://storage.googleapis.com/pub/gsutil_4.60.tar.gz
-      ```
+To enable services to run even when the `yugabyte` user is not logged in, run the following commands as the yugabyte user:
 
-  - For airgapped installations, copy `/opt/third-party/gsutil_4.60.tar.gz` from the YugabyteDB Anywhere node, as follows:
-
-      ```sh
-      cd /usr/local
-      sudo tar xvfz gsutil_4.60.tar.gz
-      sudo ln -s /usr/local/gsutil/gsutil /usr/local/bin/gsutil
-      ```
-
-## Set crontab permissions
-
-YugabyteDB Anywhere supports performing YugabyteDB liveness checks, log file management, and core file management using cron jobs.
-
-Note that sudo is required to set up this service.
-
-If YugabyteDB Anywhere will be using cron jobs, ensure that the `yugabyte` user is allowed to run crontab:
-
-- If you are using the `cron.allow` file to manage crontab access, add the `yugabyte` user to this file.
-- If you are using the `cron.deny` file, remove the `yugabyte` user from this file.
-
-If you are not using either file, no changes are required.
-
-<!--
-
-##### Manage liveness checks, logs, and cores
-
-YugabyteDB Anywhere supports performing YugabyteDB liveness checks, log file management, and core file management using cron jobs or systemd services.
-
-**Sudo is required to set up these services**
-
-If YugabyteDB Anywhere will be using **cron jobs**, make sure the yugabyte user is allowed to run crontab. If you're using the cron.allow file to manage crontab access, add the yugabyte user to this file. If you're using the cron.deny file, remove the yugabyte user from this file.
-
-YugabyteDB Anywhere **systemd services** to perform the monitoring operations mentioned above, then make sure ...
--->
+```sh
+loginctl enable-linger yugabyte
+vi ~/.bashrc
+export XDG_RUNTIME_DIR=/run/user/$(id -u yugabyte)
```

Comment:
I think we should just say add export XDG_RUNTIME_DIR=/run/user/$(id -u yugabyte) to /home/yugabyte/.bashrc . Don't need to specifically include the vi they can edit however they want.